driver/mloPoolingHost.hpp

```diff
@@ -110,7 +110,7 @@ bool mloPoolingForwardRunHostAndVerify(int pooling_method,
                                        size_t* mask_ptr,
                                        Index* mask_gpu,
                                        _Tcheck allowedEps,
-                                       int index_position = 0)
+                                       int index_position = 1)
```

Comment:
What does this change do?

Reply:
Previously the global image index was temporarily assigned to 0. In this PR, it is changed to 1 to be consistent with the value of miopenPoolingWorkspaceIndexMode_t.